indexer/beacon/epochstats_ssz.go

```diff
@@ -1,5 +1,5 @@
 // Code generated by fastssz. DO NOT EDIT.
-// Hash: eaeba007a5bccb81a6437f4146d03f627f52792f537b2b8fee9df48555afc8c5
+// Hash: faec4c530c798a99a2a7b61d4d7404c9e8aa698d1e1ab942211f1b3720248a71
```

Comment:
lol, I'm actually really curious how you re-generated this file with a new hash?
it looks fine on first glance, but there are various spots missing:
```go
// line 254:
size += len(e.ActiveValidators) * 6  // 6 was the old size, should be 12 with your change
```
I guess it's been a manual update?